Week2/todo_app/program.js

```diff
@@ -62,11 +181,46 @@ dbConnection.connect(function(err) {
     console.log('connected as id ' + dbConnection.threadId);
 
     const todoModel = new TodoModel(dbConnection);
+
+
     todoModel.load(function(err, todoItems) {
         if(err) {
             console.log("error loading TODO items:", err);
         }
 
         console.log("existing todo items:", todoItems);
     });
+    
+    
+
+    // Create new todo item :
+    const newItemText = 'Going to Utrecht';
+    const newItem = [newItemText, true, 4];
+    todoModel.create(newItem, (err) => {
+        err ? console.log("Error by adding new item", err) : console.log("New todo-item added!");
+    });
+   
+    const toUpdateId = 44;
+    const toUpdateText = "Beginning with ReactJS week1 + homework";
+    todoModel.update(toUpdateId, toUpdateText, (err) => {
+        err ? console.log("Error by updating item", err) : console.log("todo-item updated!");
+    });
+    
+    const toDeleteId = 51;
+    todoModel.delete(toDeleteId, (err) => {
+        err ? console.log("Error by deleting item", err) : console.log("todo-item deleted!");
+    });
+
+    todoModel.tagTodoItem(44, 2, (err) => {
+        err ? console.log("Error by adding new tag", err) : console.log("New todo-item tag added!");
+    });
+
+    todoModel.untagTodoItem(44, 2, (err) => {
+        err ? console.log("Error by deleting the tag", err) : console.log("tag deleted !");
+    });
+    
+    todoModel.markCompleted(43, (err) => {
```

Comment:
loved the tests, you can query the db on the mysql terminal to see that the db ends up populated like you intended 👍